RaccoonPG.py

```
@@ -0,0 +1,389 @@
+        self.inCombat = True
+
+    def increase_health(self, value):
+        self.HP += value
+
+    def reduce_health(self, enemy):
+        damage = Animal.damage_formula(self, enemy, POWER, LVL, ATTK, CRIT)
+        self.HP -= damage
+        if self.HP <= 0:
+            self.inCombat == False
+
+    def lvl_up(self, EXP, EXPcap, LVL):
```

Ruff (0.14.0)

Unused method argument: `EXP`

(ARG002)